scripts/035-bzip2_dbglogger.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+#
+# bzip2 by cadcdev
+# dbglogger by Bucanero
+
+## Download the source code.
+wget --no-check-certificate https://github.com/bucanero/psl1ght-libs/tarball/master -O psl1ght-libs.tar.gz 
```

Comment:
Now there's a new repo for the `dbglogger` library.

Can you update the script so it downloads from https://github.com/bucanero/dbglogger ?